build_runner/lib/src/generate/build_result.dart

```diff
@@ -56,6 +64,16 @@ enum BuildStatus {
   failure,
 }
 
+/// The type of failure
+///
+/// Values correspond to proposed exitCode.
+class FailureType {
+  static const general = const FailureType(1);
+  static const cantCreate = const FailureType(73);
+  final int value;
```

Comment:
nit: rename exitCode?

Reply:
+1

Reply:
package:io doesn't expose those values but I have updated the name to exitCode.